src/Track.cpp

```diff
@@ -864,7 +882,7 @@ Track *TrackList::GetPrev(Track * t, bool linked) const
             if (linked) {
                prev = getPrev( node );
                if( !isNull( prev ) &&
-                   !(*node.first)->GetLinked() && (*node.first)->GetLink() )
+                   !(*node.first)->HasLinkedTrack() && (*node.first)->GetLeaderTrack() )
```

Comment:
I do like the name changes making this line read less strangely.

That's all for reviewing this PR.